src/gwt/src/org/rstudio/core/client/widget/SimplePanelWithProgress.java

```diff
@@ -71,6 +71,11 @@ public void onResize()
       if (getWidget() instanceof RequiresResize)
          ((RequiresResize)getWidget()).onResize();
    }
+   
+   public void setInheritEditorTheme(boolean inherit)
+   {
+      loadProgressPanel_.setInheritEditorTheme(inherit);
```

Comment:
This lets us avoid a flashing black background for dialogs where the background is white, but a dark editor theme is being used.